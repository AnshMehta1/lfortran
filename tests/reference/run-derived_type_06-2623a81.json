{
    "basename": "run-derived_type_06-2623a81",
    "cmd": "lfortran --no-color {infile}",
    "infile": "tests/errors/derived_type_06.f90",
    "infile_hash": "002cfbfb5901ee3c9f0d5e33285543863528f17c4b8096e1a0e49134",
    "outfile": null,
    "outfile_hash": null,
    "stdout": null,
    "stdout_hash": null,
    "stderr": "run-derived_type_06-2623a81.stderr",
    "stderr_hash": "13dc71322f8815413728c8a98468757fda295f39e22ab066b2874734",
    "returncode": 1
}