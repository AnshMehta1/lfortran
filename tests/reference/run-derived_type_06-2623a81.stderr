semantic error: Two or more part references with non-zero rank must not be specified.
  --> tests/errors/derived_type_06.f90:14:5
   |
14 |     main_arr%arr_2%num = 22
   |     ^^^^^^^^^^^^^^^^^^ 
